spec/core/integration/EnvSpec.js

```diff
@@ -1400,7 +1400,7 @@ describe("Env integration", function() {
         status: 'passed'
       }));
 
-      expect(reporter.specDone).toHaveBeenCalledWith(jasmine.objectContaining({
+      expect(reporter.specDone).not.toHaveBeenCalledWith(jasmine.objectContaining({
```

Comment:
Does this mean that we'd no longer be reporting pending specs? That would be a problem. We're looking to improve reporting of pending specs (see e.g. #1418), not remove it.

Reply:
Currently, yes. However, change could probably be amended such that `specDone` is still called for pending specs, but `beforeAll` is not. I am not very familiar with this code, but perhaps you would be able to make such an improvement.

Reply:
I like the idea of making sure we don't run setup code that we don't need, but we definitely don't want to merge this pull request if it is going to break reporting for pending specs.